src/content/developers/tutorials/executing-and-interacting-with-a-smart-contract/index.md

```diff
@@ -0,0 +1,247 @@
+---
+title: 💻 Executing and Interacting with a Smart Contract (Part 2/4 of Hello World dApp Series)
+description: Step by step guide on interacting with a deployed Ethereum smart contract by updating a smart contract variable.
+author: "nstrike2"
+tags:
+  [
+    "solidity",
+    "hardhat",
+    "truffle",
+    "alchemy",
+    "smart contracts",
+    "getting started",
+    "deploying"
+  ]
+skill: beginner
+lang: en
+sidebar: true
+published: 2021-10-06
+---
+# 💻 Executing and Interacting with a Smart Contract
```

Comment:
We don't currently have support for using Emojis in headings (we use Twemoji to ensure emojis are consistent across browsers but it breaks the table of contents include them in headers). Can we remove them across these tutorials, please?

```suggestion
# Executing and Interacting with a Smart Contract
```

Reply:
Also, could you please ensure all headers have header ids

e.g. `# Executing and Interacting with a Smart Contract {#executing-and-interacting}`